cli/tests/unit_node/worker_threads_test.ts

```diff
@@ -133,6 +133,20 @@ Deno.test({
   },
 });
 
+Deno.test({
+  name: "[worker_threads] worker thread with type module",
+  fn() {
+    try {
```

Comment:
nit: Is this 'try' block necessary? This looks obfuscating the actual error by swallowing it

Reply:
Should we allow throwing of actual error? I thought of doing this so fail happen via assert and not error. Will change this.

Reply:
It should be fine to throw error from a test case. It's detected by the test runner (also we know the details of that error)